src/main/java/controller/AttendanceController.java

```diff
@@ -0,0 +1,94 @@
+package controller;
+
+import static util.Convertor.changeStandardDate;
+
+import constants.SelectionOption;
+import domain.AbsenceLevel;
+import domain.Crew;
+import domain.Crews;
+import domain.History;
+import dto.AbsenceCrewDto;
+import dto.AbsenceCrewsDto;
+import dto.HistoriesDto;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import view.InputView;
+import view.OutputVIew;
+
+public class AttendanceController {
+
+    private final OutputVIew outputVIew;
+    private final InputView inputView;
+    private final Crews crews;
+
+    public AttendanceController(OutputVIew outputVIew, InputView inputView, Crews crews) {
+        this.outputVIew = outputVIew;
+        this.inputView = inputView;
+        this.crews = crews;
+    }
+
+    public void start() {
+        SelectionOption answer;
+        do {
+            answer = inputView.getMenu();
+            startMenu(answer);
+        } while (!(answer == SelectionOption.QUIT));
```

Comment:
조건문을 `!=` 대신에 `!(...)`로 두신 이유가 있을까요? 
```suggestion
        } while (answer != SelectionOption.QUIT));
```

Reply:
앗..! 아무 이유 없습니다! 로키 코드처럼 `!=` 를 사용하는게 의미 전달이 확실할거 같아요!